Sources/Sentry/include/SentryNSProcessInfoWrapper.h

```diff
@@ -4,6 +4,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface SentryNSProcessInfoWrapper : NSObject
 
+@property (class, readonly) SentryNSProcessInfoWrapper *shared;
```

Comment:
Is there any way we can do what you need without introducing more singletons? Can you let us know why it's better/necessary to convert this to a singleton?

Reply:
Yes its possible, by creating another instance of SentryNSProcessInfoWrapper.
`SentryNSProcessInfoWrapper` consist of 3 functions that pass forward the call to another API, it does not have any variables nor any behaviour. Why should we allocated memory to hold reference to it every time we need?